doc/manpages/faq.asciidoc

```diff
@@ -133,6 +133,16 @@ The shell scope allows users to spawn any interpreter they want, for a minimal
 cost in terms of performance, it is therefore the reason why it's the only
 one available by default.
 
+What shell is used to expand `%sh{}` scopes ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The server expands shell scopes using `/bin/sh` (which is, in most
```

Comment:
I just tried to show that `/bin/sh` isn't necessarily a shell by itself (as is the case in FreeBSD and OpenBSD), but can be a link to another shell (as is the case in Archlinux, Debian, Gentoo).

I could have sworn there was a hardcoded `/bin/sh` in the shell manager at some point, which mislead me, so I need to rewrite this entire answer.